deps/openssl/doc/FIPS.md

```diff
@@ -0,0 +1,19 @@
+# Building io.js with FIPS-compliant OpenSSL
```

Comment:
could we have a link to this from the README? Perhaps at the bottom of the build section:

```
See also: [Building io.js with FIPS-compliant OpenSSL](./deps/openssl/doc/FIPS.md)
```